030_Data/20_Exists.asciidoc

```diff
@@ -33,6 +31,4 @@ Content-Type: text/plain; charset=UTF-8
 Content-Length: 0
 --------------------------------------------------
 
-Of course, just because a document didn't exist when you checked it, doesn't
-mean that it won't exist a millisecond later: another process might create the
-document in the meantime.
+当然，当你检查一个文档时它不存在，不意味着一毫秒之后它不存在：也许在同一时间另一个进程就创建了这个文档。
```

Comment:
Of course, just because a document didn't exist when you checked it, doesn't mean that it won't exist a millisecond later:
这里可不可以这样翻译：
当然，这只是说你检查一个文档的那个时候它不存在，不意味着一毫秒之后它不存在：